about.html

```diff
@@ -31,44 +31,54 @@
       </nav>
-      <nav class="menu-items">
-      <a href="#ai-logo" onclick="closeNav()">Angel Investor Logo</a>
-      <a href="#time-cards" onclick="closeNav()">Timeline</a>
-      <a href="#partner" onclick="closeNav()">Partner</a>
-      <a href="index.html" onclick="closeNav()">Home</a>
-      </nav>
+      <div class="w-100 d-nav align-items-center">
+        <img class=" col-3 mx-auto logo menu-items align-self-center pb-2" src="src/ailogo.png" alt="">
+        <nav class="menu-items col-8 align-content-center m-0 h-100 me-5">
+          <a href="#ai-logo" onclick="closeNav()">Angel Investor Logo</a>
+          <a href="#time-cards" onclick="closeNav()">Timeline</a>
+          <a href="#partner" onclick="closeNav()">Partner</a>
+          <a href="index.html" onclick="closeNav()">Home</a>
+          <button class="h-50 align-self-center secondary head-btn">AI
+            Campaign</button>
+        </nav>
+      </div>
+
     </nav>
   </header>
   <!-- Header -->
-  <section class="header-container p-2">
+  <section class="header-container p-2 p-md-5">
     <h3 class="text-center">"Hello! Entrepreneurs"</h3>
-      <h1 class="text-center pb-3">Angel Investor Business Summit 2022</h1>
-      <p class="text-center bg-white card p-3">Lorem ipsum dolor sit amet consectetur adipisicing elit. Qui dolore quas temporibus! Maiores tempora
-        ullam aperiam cupiditate excepturi eligendi ducimus iusto? Qui aliquam illum velit veritatis est eos
-        corporis nisi.
-      </p>
-      <p class="text-center mb-0 ps-4 pe-4"><small>Please contact us via Email for any further questions about AI Business Summit 2022</small></p>
-      <p class="text-decoration-underline text-center fw-bold">aisummit2022@aiinvestor,com</p>
+    <h1 class="text-center pb-3">Angel Investor Business Summit 2022</h1>
+    <p class="text-center bg-white card p-3">Lorem ipsum dolor sit amet consectetur adipisicing elit. Qui dolore quas
+      temporibus! Maiores tempora
+      ullam aperiam cupiditate excepturi eligendi ducimus iusto? Qui aliquam illum velit veritatis est eos
+      corporis nisi.
+    </p>
+    <p class="text-center mb-0 ps-4 pe-4"><small>Please contact us via Email for any further questions about AI Business
+        Summit 2022</small></p>
+    <p class="text-decoration-underline text-center fw-bold">aisummit2022@aiinvestor,com</p>
   </section>
   <!-- logo section -->
-  <section class="logo-container bg-white p-5" id="ai-logo" >
+  <section class="logo-container bg-white p-5" id="ai-logo">
     <h3 class="featured-header text-center">Angel Investor Business Summit 2022 Logo</h3>
     <p class="divider text-center">_________</p>
-    <p class="mb-5 text-center">The logo of Angel Investor Business Summit 2022 was decided through the logo competition that was held on May 5th
+    <p class="mb-5 text-center">The logo of Angel Investor Business Summit 2022 was decided through the logo competition
+      that was held on May 5th
     </p>
```

Comment:
- [ ] Your background should be images with a certain background color on it ,
Your design 
![image](https://user-images.githubusercontent.com/43877689/181494139-1871e4bd-9592-46cc-9366-15dc8e0c1fdb.png)

Behance design 

![image](https://user-images.githubusercontent.com/43877689/181494260-98d34273-3fba-4fc7-b7d2-00c73ae045d9.png)